packages/slice-machine/src/features/onboarding/useOnboardingExperiment.ts

```diff
@@ -1,6 +1,6 @@
 import { useExperimentVariant } from "@/hooks/useExperimentVariant";
 
 export const useOnboardingExperiment = () => {
-  const variant = useExperimentVariant("slice-machine-onboarding");
+  const variant = useExperimentVariant("slicemachine-onboarding");
```

Comment:
❓ #question

Why this rename? It was because of the non-consistent naming with `-`?
I still see `slice-machine-onboarding` in Amplitude, should it be removed, deactivated?

Reply:
Yeah I've been meaning to discuss this. It was a typo, Levi pointed out that all the other slice machine ones didn't have a dash, but since that version was released, I'm not sure it was wise to do so. I was planning on archive it after releasing.

Reply:
Oh I see, you can test in stage anyway the thing you want to do, to ensure it doesn't cause a problem while testing the prod slice machine version that target it.

Reply:
To test in stage you need `SM_ENV=staging npm run slicemachine`